src/dataflow/graph/graph.ts

```diff
@@ -267,6 +296,16 @@ export class DataflowGraph {
 		return this
 	}
 
+	/**
+	 * Adds a read edge for simple testing.
+	 * 
+	 * @param from - Vertex/Node id
+	 * @param to   - see from
+	 * @param when - always (default), or maybe
+	 */
+	public reads(from: NodeId, to: NodeId, when: DataflowGraphEdgeAttribute = 'always') {
```

Comment:
Like `uses`, `reads` is a simpler/more concise version of `addEdge`.

Reply:
Of course, other versions for calls, arguments, defined-by etc. are still required.